scripts/select-java

```diff
@@ -26,10 +26,18 @@ function select_java_fedora() {
 }
 
 function select_java_others() {
-    javaver=$(/usr/bin/java -version 2>&1|head -n1|cut -f 3 -d " ")
-    if [[ "$javaver" =~ ^\"1.8.0 || "$javaver" =~ ^\"11.0. ]]; then
-        echo /usr/bin/java
-    fi
+    local java
+    local javaver
+    java=$1
+    shift
+    javaver=$($java -XshowSettings:properties -version 2>&1 | awk -F' = ' '/java.specification.version/ {print $NF}')
+    case "$javaver" in
+    1.8|11)
+        echo $java
+        ;;
+    *)
+        ;;
+    esac
 }
```

Comment:
Why is everything so complicated? Can't we check /usr/lib/jvm?

Reply:
You can bring containers to offline deployments...

Reply:
these steps were inherited from 96adf0ef51ad49b6abb98867f73cf69705f5f3e3. by then, we didn't check `/usr/lib/jvm`,  but now that we are here, i think we can drop it in favor of checking the ones under in `/usr/lib/jvm/`. like
```bash
for expected_java_version in 11 1.8; do
  for java_home in /usr/lib/jvm/*; do
    java=$java_home/bin/java
    javaver=$($java -XshowSettings:properties -version 2>&1 | awk -F' = ' '/java.specification.version/ {print $NF}')
    if [ "$javaver" = $expected_java_version ]; then
        echo $java
        return
    fi
  done
done
```